backend/postgres/schema.sql

```diff
@@ -0,0 +1,57 @@
+CREATE SCHEMA IF NOT EXISTS durabletask;
+
+CREATE TABLE IF NOT EXISTS durabletask.Instances (
```

Comment:
More of a nit, but... Postgres folds all identifiers so they're stored as lowercase (you can verify that with pgadmin, for example). So, "Instances" and "instances" are the same identifier. Because of that, more often than not in Postgres people tend to use snake_case for identifiers.